resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientApplication.scala

```diff
@@ -21,9 +21,11 @@ import java.util.{Collections, UUID}
 import java.util.Properties
 
 import io.fabric8.kubernetes.api.model._
-import io.fabric8.kubernetes.client.KubernetesClient
+import io.fabric8.kubernetes.client.{KubernetesClient, Watch}
+import io.fabric8.kubernetes.client.Watcher.Action
 import scala.collection.mutable
 import scala.util.control.NonFatal
+import util.control.Breaks._
```

Comment:
@jkleckner Apache Spark community always use absolute path. Please fix this.
> Always import packages using absolute paths (e.g. scala.util.Random) instead of relative ones (e.g. util.Random).

Reply:
Done.
Note that the original merged patch didn't do this but was fixed later on master in  527cd3fc3aac40f84ba8eee291e1a955e03f7665

Reply:
Ok, rebase fixed integration test.  Ok to resolve this thread?